ui_testing/pages/testplan_page.py

```diff
@@ -117,20 +118,29 @@ def get_test_unit_category(self):
         return self.base_selenium.get_text(element='test_plan:test_unit_category')
 
 
-    def create_new_test_plan(self, name='', material_type='', article='', test_unit='', save=True, **kwargs):
+    def create_new_test_plan(self, name='', material_type='', article='', test_unit='', save=True,multiple_material_types=False, **kwargs):
         self.info(' Create new test plan')
```

Comment:
you can make material_type=[] not sting and remove this tag :multiple_material_types